.travis.yml

```diff
@@ -220,3 +220,56 @@ jobs:
       script:
       # run illuminatio e2e tests
       - python setup.py test --addopts="-m e2e"
+
+    - stage: build
```

Comment:
Could you add a small comment above the step what they are doing and when they are executed ?